### ISSUE TITLE
Enable Prometheus metrics

### ISSUE THREAD

**Issue by cbcoutinho** [jellyfin/jellyfin-helm#28]
I would like to simplify metrics collection for Jellyfin using this Helm chart, as Jellyfin currently supports exposing Prometheus metrics, although this functionality is currently disabled by default in the upstream image.

This is how I'm currently working around the limitation in the helm chart, based on #27; however, I'm open to improvements.

```yaml
# values.yaml
  image:
    command:
      - bash
      - '-c'
      - >-
        sed 's,<EnableMetrics>false</EnableMetrics>,<EnableMetrics>true</EnableMetrics>,' -i /config/config/system.xml &&
        /jellyfin/jellyfin
```

As well as creating a ServiceMonitor:

```yaml
apiVersion: monitoring.coreos.com/v1
kind: ServiceMonitor
metadata:
  name: jellyfin-svc-monitor
  namespace: jellyfin
  labels:
    release: observability
spec:
  namespaceSelector:
    matchNames:
      - jellyfin
  selector:
    matchLabels:
      app.kubernetes.io/name: jellyfin
  endpoints:
    - port: http
      path: /metrics
      interval: 15s
```

**Comment by Supporterino** [comment]
Hi @cbcoutinho,

I took your issue as a general idea and modified it to be modular inside the helm chart until the upstream docker container provides a better way to enable metrics. Check #39 for status

**Comment by Supporterino** [comment]
@crobibero this issue can be closed as it is implemented in the latest version

**Comment by samcannell** [comment]
The way this is written means that Jellyfin fails to start the first time (i.e. when system.xml does not already exist):

```
$ kubectl logs -n jellyfin deploy/jellyfin
sed: can't read /config/config/system.xml: No such file or directory
$
```

Using ';' in the command instead of '&&' is an improvement -- the sed error still appears, but Jellyfin still starts; the downside of course is that metrics won't be available until after the Jellyfin pod is killed and started a second time:

```
$ kubectl get deploy -n jellyfin jellyfin -o jsonpath='{.spec.template.spec.containers[0].command}{"\n"}'
["bash","-c","sed 's,\u003cEnableMetrics\u003efalse\u003c/EnableMetrics\u003e,\u003cEnableMetrics\u003etrue\u003c/EnableMetrics\u003e,' -i /config/config/system.xml ; /jellyfin/jellyfin"]
$
```

```
$ kubectl logs -n jellyfin deploy/jellyfin
sed: can't read /config/config/system.xml: No such file or directory
[18:17:19] [INF] [4] Main: Jellyfin version: 10.10.3
[18:17:19] [INF] [4] Main: Environment Variables: ["[JELLYFIN_CACHE_DIR, /cache]", "[JELLYFIN_FFMPEG, /usr/lib/jellyfin-ffmpeg/ffmpeg]", "[JELLYFIN_PORT_8096_TCP_PORT, 8096]", "[JELLYFIN_SERVICE_HOST, 172.24.141.151]", "[JELLYFIN_PORT_8096_TCP_PROTO, tcp]", "[JELLYFIN_PORT_8096_TCP_ADDR, 172.24.141.151]", "[JELLYFIN_SERVICE_PORT, 8096]", "[JELLYFIN_CONFIG_DIR, /config/config]", "[JELLYFIN_LOG_DIR, /config/log]", "[JELLYFIN_PORT_8096_TCP, tcp://172.24.141.151:8096]", "[JELLYFIN_WEB_DIR, /jellyfin/jellyfin-web]", "[JELLYFIN_DATA_DIR, /config]", "[JELLYFIN_SERVICE_PORT_SERVICE, 8096]", "[JELLYFIN_PORT, tcp://172.24.141.151:8096]"]
[18:17:19] [INF] [4] Main: Arguments: ["/jellyfin/jellyfin.dll"]
[18:17:19] [INF] [4] Main: Operating system: Debian GNU/Linux 12 (bookworm)
[...]
```

**Comment by cbcoutinho** [comment]
I've noticed that there has been an increase in activity related to this request since I first raised it, which also resulted in an issue for new installations #54  and the related fix in #65.

I think using startup commands in general aren't a great idea - I've moved several to sidecar containers but this particular feature has nagged me for some time. This weekend I took a crack at enabling monitoring via environmental variables in the jellyfin server itself.

For anyone interested in taking a look at the PR please see: https://github.com/jellyfin/jellyfin/pull/14052
